cli/src/pcluster/aws/aws_resources.py

```diff
@@ -242,6 +242,10 @@ def supported_usage_classes(self):
             supported_classes.append("ondemand")
         return supported_classes
 
+    def ec2memory_size_in_mib(self):
+        """Return the amount of memory in MiB."""
+        return self.instance_type_data.get("MemoryInfo", {}).get("SizeInMiB")
```

Comment:
@demartinofra and I discussed about this. We decided to default to 1 (this can happen only if memory-based scheduling is disabled and instance_types_data does not include memory information for the requested instance type).